stdlib/public/Platform/glibc.modulemap.in.gyb

```diff
@@ -37,28 +37,29 @@ module SwiftGlibc [system] {
       export *
     }
 
-    // glibc specific
+% if CMAKE_SDK != "FREEBSD":
     module features {
       header "@GLIBC_INCLUDE_PATH@/features.h"
       export *
     }
+% end
 
     module fenv {
       header "@GLIBC_INCLUDE_PATH@/fenv.h"
       export *
     }
-    
+
     // note: supplied by compiler
     // module float {
     //   header "@GLIBC_INCLUDE_PATH@/float.h"
```

Comment:
Now that we are using gyb, could we port the `@`-variables to use gyb as well?  Would save some complexity in CMake rules.